packages/frontend/core/src/desktop/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -301,19 +301,19 @@ const DetailPageImpl = memo(function DetailPageImpl() {
         </ViewSidebarTab>
       )}
 
-      <ViewSidebarTab tabId="properties" icon={<PropertyIcon />}>
+      <ViewSidebarTab tabId="properties" icon={<PropertyIcon />} scrollable>
         <DocPropertySidebar />
       </ViewSidebarTab>
```

Comment:
```suggestion
      <ViewSidebarTab tabId="properties" icon={<PropertyIcon />>
        <Scrollable.Root>
          <Scrollable.Viewport><DocPropertySidebar /></Scrollable.Viewport>
          <Scrollable.Scrollbar />
        </Scrollable.Root>
      </ViewSidebarTab>
```

Would it be more reasonable if we put the scroll bar inside?
Currently it seems that all tabs share one scroll bar.

Reply:
All tabs should be scrollable by default, except AI

Reply:
Otherwise we have to wrap a Scrollable for each tab?(but it could avoid mounting and unmounting components when switching tab)